scripts/base/protocols/ssl/main.zeek

```diff
@@ -499,6 +499,15 @@ event analyzer_confirmation_info(atype: AllAnalyzers::Tag, info: AnalyzerConfirm
 		}
 	}
 
+event analyzer_violation_info(atype: AllAnalyzers::Tag, info: AnalyzerViolationInfo) &priority=5
+	{
+	if ( atype == Analyzer::ANALYZER_SSL || atype == Analyzer::ANALYZER_DTLS )
+		{
+		# analyzer errored out; prevent us from trying to remove it later
+		delete info$c$ssl$analyzer_id;
```

Comment:
Hmm.. could also see invoking `disable_analyzer()` with `err_if_no_conn=F` instead to suppress the error message?

There's an `analyzer_violation_info()` handler a few lines below with the same priority that ends-up in `finish()` and attempts to call `disable_analyzer()`, so the interaction of these two currently might not be obvious.

Reply:
Oh, I am blind. You are right - there are two handlers. I will fold it into the second one - that will also make this definitely deterministic.

I like that slightly better than suppressing the error message, but I guess one can have different opinions about that :)